cross_dock/views.py

```
@@ -73,45 +99,39 @@ def process_file(request):
-                    task = CrossDockTask.objects.get(id=task_id)
-                    task.mark_as_success(output_url)
-
-            finally:
+                # Clean up the uploaded file
                 try:
                     os.remove(file_path)
-                    logger.info(f"Removed temporary upload file: {file_path}")
-                except Exception as e:
-                    logger.warning(f"Failed to remove temporary upload file {file_path}: {e}")
+                except Exception:
+                    pass
```

Ruff (0.11.9)

Use `contextlib.suppress(Exception)` instead of `try`-`except`-`pass`

Replace with `contextlib.suppress(Exception)`

(SIM105)